Task 3/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Show Github users</title>
+    <!-- Custom CSS -->
+    <link rel="stylesheet" href="./style.css" />
+    <!-- Custom JS -->
+    <script defer src="./script.js"></script>
+  </head>
+  <body>
+    <header>
+      <!-- Header code-->
+    </header>
+
+    <main>
+      <div class="wrapper">
+        <div class="btn-container">
+          <button onclick="fetchData()" id="btn">Show Users</button>
```

Comment:
Ne pas jus pirmą matau tokią praktiką, kurios niekas jūsų nemokė. Inline JS yra bloga praktika ir to niekas seniai nenaudoja. Vienas iš argumentų, kad tokį kodą būtų sunku palaikyti, nes reikia bėgioti tarp HTML ir JS, kad eitų atsekti kas prie ko jungiama. Neretai HTML būna didelės apimties, kad tik dar labiau apsunkina situacija. Kitose užduotyse tokių atvejų nekomentuosiu, bet atsižvelgsiu vertinant.